deb/ar.go

```diff
@@ -18,7 +18,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE. }}} */
 
-package deb // import "pault.ag/go/debian/deb"
```

Comment:
Please retain these, if we need to remove them, let's start a new PR removing them repo-wide. I'm guessing this is due to the package replacement that's still intact below

Reply:
Ok--

Pro: if you remove them, then anyone can host the same package elsewhere and not have GO throw alerts that it isn't in the right place
Con: the control of the sourcecode baseline can help with keeping the changes centralized.

Reply:
Yeah totally. We can discuss the merits of this one in another PR, this style is more common in pre go.mod code so I'm not opposed on it's face. Let's keep that change out of this PR if you don't mind though!